db/migrate/20191016144506_copy_latest_provider_enrichment_data_to_provider.rb

```diff
@@ -0,0 +1,37 @@
+class CopyLatestProviderEnrichmentDataToProvider < ActiveRecord::Migration[6.0]
+  def up
+    say_with_time "copy provider enrichment to provider" do
+      enrichment_fields = %w[train_with_us
+                             train_with_disability
+                             accrediting_provider_enrichments
+                             last_published_at]
+
+      contact_fields = %w[email
+                          telephone
+                          address1
+                          address2
+                          address3
+                          address4
+                          postcode
+                          region_code
+                          website]
+
+      Provider.includes(:enrichments, :recruitment_cycle).each do |provider|
+        latest_created_at = provider.enrichments.max_by { |e| [e.created_at, e.id] }
```

Comment:
`e.created_at` or `e.updated_at` ? Feels like `e.updated_at` will get us the most recent one the user has edited

Reply:
amended to use `updated_at` instead